packages/micro.rb

```diff
@@ -22,6 +18,8 @@ class Micro < Package
     source_sha256 'a50e405d3d09d58f6b2c182429c18537a05f317dc0c3c9cb834b3271362e4781'
   end
 
+  no_compile_needed
+  
```

Comment:
Rubocop complaining about trailing whitespace.

Can we add a `rubocop -c .rubocop.yml -A .` to whatever pre-commit hooks we have? (Not needed for this PR... Just saying that it seems that rubocop can just do these cleanups for us...)